README.md

```diff
@@ -16,12 +16,28 @@ And then to building the `tsoa` routes:
 npm run tsoa:build
 ```
 
+Followed by database migrations
+
+```bash
+npm run db:migrate
+# seeding data
+npm run db:seed
```

Comment:
this isn't part of the migrations. Can we document it in a separate block please